composer.json

```diff
@@ -29,6 +29,7 @@
 		}
 	},
 	"require": {
-		"phpunit/phpunit": "9.4.4"
+		"phpunit/phpunit": "9.4.4",
```

Comment:
I know it was here before, but do we really need PHPUnit in the `require` and include it to the release together with IPLib?